pyproject.toml

```diff
@@ -32,7 +32,7 @@ typer = "^0.9.0"
 types-requests = "^2.31.0.20240125"
 types-python-dateutil = "^2.8.19.20240106"
 # TODO: Use pypi package once released.
-prediction-market-agent-tooling = {git="https://github.com/gnosis/prediction-market-agent-tooling.git", branch="main"}
+prediction-market-agent-tooling = {git="https://github.com/gnosis/prediction-market-agent-tooling.git", branch="peter/add-sorting"}
```

Comment:
Can move back to main now :)

Reply:
I had to specify exact commit, because latest main is no longer compatible with this. It will be updated to pypi in another PR.